packages/cdc-react/src/components/Checkbox/Checkbox.tsx

```diff
@@ -0,0 +1,46 @@
+// Checkbox.tsx
+import { ChangeEvent, ReactNode, useState } from "react";
+import "./Checkbox.scss";
+
+type CheckboxProps = {
+  id: string;
+  label?: string;
+  description?: string;
+  checkedBox?: boolean;
+  onChange?: (event: ChangeEvent<HTMLInputElement>) => void;
```

Comment:
Looks like this isn't getting called anywhere.  I'm guessing it is intended to be used to get the current state out of the component.